core/templates/dev/head/components/AnswerGroupEditorDirective.js

```diff
@@ -41,15 +40,22 @@ oppia.directive('answerGroupEditor', [
         '$scope', 'stateInteractionIdService', 'ResponsesService',
         'EditorStateService', 'AlertsService', 'INTERACTION_SPECS',
         'RULE_TYPE_CLASSIFIER', 'RuleObjectFactory',
+        'explorationCorrectnessFeedbackService',
         function(
             $scope, stateInteractionIdService, ResponsesService,
             EditorStateService, AlertsService, INTERACTION_SPECS,
-            RULE_TYPE_CLASSIFIER, RuleObjectFactory) {
+            RULE_TYPE_CLASSIFIER, RuleObjectFactory,
+            explorationCorrectnessFeedbackService) {
           $scope.rulesMemento = null;
           $scope.activeRuleIndex = ResponsesService.getActiveRuleIndex();
           $scope.editAnswerGroupForm = {};
           $scope.answerGroupIsDefault = ($scope.rules !== null);
           
+          $scope.isCorrectnessFeedbackEnabled = function() {
+            return (
+              explorationCorrectnessFeedbackService.isEnabled());
```

Comment:
I guess this can now go on the previous line (and the surrounding parens removed)?

Reply:
Done!